third_party/2and3/sqlalchemy/engine/base.pyi

```diff
@@ -1,20 +1,28 @@
-from typing import Any, List, Tuple
+from typing import Any, Iterable, Iterator, List, Mapping, Optional, Tuple
 
 # Dummy until I figure out something better.
 class Connectable:
```

Comment:
This class actually has numerous additional methods and it lives in `interface.pyi`. I'm not sure how valuable it is to add those additional methods, but in general it might be worse to have incomplete stubs (missing important functions) than no stubs, because with incomplete stubs people will get false positive errors about missing functions. It sounds like that's currently happening with SQLAlchemy (#974).